### PR TITLE
Update node info description in node.md

### PR DESCRIPTION
**Pull request by feihujiang**
### Node Info
General information about the node, for instance ..., "kubernetes
version" ... changed to "kubernetes
version (kubelet version, kube-proxy version)"

### CODE REVIEW

**Comment by feihujiang** [comment]
@marekbiskup Could you please review this PR?

File: docs/node.md [hunk 1]
```diff
@@ -61,8 +61,8 @@ Describes the resources available on the node: CPUs, memory and the maximum
 number of pods that can be scheduled on this node.
 
 ### Node Info
-General information about the node, for instance kernel version, kubernetes
-version, docker version (if used), OS name. The information is gathered by 
+General information about the node, for instance kernel version, kubelete version,
+kube-Proxy version, docker version (if used), OS name. The information is gathered by 
 Kubernetes from the node.
```
Comment (marekbiskup) [inline docs/node.md]: it should rather be 
```kubelet version, kube-proxy version```

However, I suggest:
```kubernetes version (kubelet version, kube-proxy version)```

Or just leave it as it was before.

Comment (feihujiang) [inline docs/node.md]: What does kubernetes version mean? To me, it is confusing, and isn't precise. I think kubernetes version (kubelet version, kube-proxy version) is nice.

Comment (marekbiskup) [inline docs/node.md]: Sounds good.

**Comment by feihujiang** [comment]
@marekbiskup Sorry for the typo. I fixed

**Comment by marekbiskup** [comment]
LGTM

**Pull request merged**